src/capella/presets/mainnet.rs

```diff
@@ -0,0 +1,172 @@
+pub use crate::altair::mainnet::SYNC_COMMITTEE_SIZE;
```

Comment:
it looks like this file is not `rustfmt`'d ?

Reply:
Thanks, should be fixed now! I have "Run rustfmt on Save" enabled in my IDE but for some reasons it didn't work :/ I have rustfmt'd the entire capella directory so this should be good now.